client/components/jetpack/jetpack-product-info/regular-list.tsx

```diff
@@ -1,14 +1,55 @@
-import { FunctionComponent, ReactNode } from 'react';
+import { useRef, useEffect, type FC, type ReactNode } from 'react';
 
 type Props = {
 	items: ReactNode[];
 };
 
-const JetpackProductInfoRegularList: FunctionComponent< Props > = ( { items } ) => {
+const JetpackProductInfoRegularList: FC< Props > = ( { items } ) => {
```

Comment:
I think the FC type can be omitted:

```suggestion
const JetpackProductInfoRegularList = ( { items }: Props ) => {
```